binary_search/binary_search.py

```diff
@@ -0,0 +1,54 @@
+from random import randint
+
+
+def binary_search_iterative(array, left, right, element):
+    while left <= right:
+        mid = int(left + (right - left) / 2)
+        # indices of a list must be integer
+        if array[mid] == element:
+            return mid
+        elif array[mid] < element:
+            left = mid + 1
+        else:
+            right = mid - 1
+    return -1
+
+
+def binary_search_recursive(array, left, right, element):
```

Comment:
Can set default values to `left` and `right` as `0` and `None` respectively. If `None`, `right` can be set to `len(array)`.
```python
def binary_search_recursive(array, element, left=0, right=None):
    right = len(array) if right is None else right
    ...
```
How would it be?